imagetest/test_suites/hostkey/hostkey_pubish_test.go

```diff
@@ -0,0 +1,60 @@
+// +build cit
+
+package hostkey
+
+import (
+	"flag"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+var (
+	runtest = flag.Bool("runtest", false, "really run the test")
+)
+
+func getHostKeysFromDisk() (map[string]string, error) {
+	bytes, err := ioutil.ReadFile("/etc/ssh/ssh_host_*_key.pub")
+	if err != nil {
+		return nil, err
+	}
+	hostkeyLines := strings.Split(strings.TrimSpace(string(bytes)), "\n")
+
+	var hostkeyMap = make(map[string]string)
+	for _, hostkey := range hostkeyLines {
+		keyType := strings.Split(hostkey, " ")[0]
+		keyValue := strings.Split(hostkey, " ")[1]
+		hostkeyMap[keyType] = keyValue
+	}
+
+	return hostkeyMap, nil
+}
+
+// TestMatchingKeysInGuestAttributes validate that host keys in guest attributes match those on disk.
```

Comment:
Can you include which code is responsible for copying the keys? That way, if there's a failure, then the future reader knows where to look. For example: "validate that the guest agent copies the host keys from metadata to the disk"

Reply:
are you saying we add comments for the link of the guest agent code that we test functionality?  I add a comment to before for loop.